.github/workflows/REVIEW_GUIDELINES.md

```diff
@@ -0,0 +1,712 @@
+// Good
+
+if (! $goodCondition) {
+  throw new Exception;
+}
+
+// do work
+```
+
+
+```php
+// Bad
+
+if ($goodCondition) {
+ // do work
+}
+
+throw new Exception;
+```
+
+### Avoid else
+
+In general, `else` should be avoided because it makes code less readable. In most cases it can be refactored using early returns. This will also cause the happy path to go last, which is desirable.
+
+```php
+// Good
+
+if (! $conditionBA) {
+   // conditionB A failed
+   
+   return;
+}
+
+if (! $conditionB) {
+   // conditionB A passed, B failed
+   
+   return;
+}
+
+// condition A and B passed
+```
+
+```php
+// Bad
+
+if ($conditionA) {
+   if ($conditionB) {
+      // condition A and B passed
+   }
+   else {
+     // conditionB A passed, B failed
+   }
+}
+else {
+   // conditionB A failed
+}
+```
+
+
+### Compound ifs
+
+In general, separate `if` statements should be preferred over a compound condition. This makes debugging code easier.
+
+
+```php
+// Good
+if (! $conditionA) {
+   return;
+}
+
+if (! $conditionB) {
+   return;
+}
+
+if (! $conditionC) {
+   return;
+}
+
+// do stuff
+```
+
+```php
+// bad
+if ($conditionA && $conditionB && $conditionC) {
+  // do stuff
+}
+```
+
+
+
+## Comments
+
+Comments should be avoided as much as possible by writing expressive code. If you do need to use a comment, format it like this:
+
+```php
+// There should be a space before a single line comment.
+
+/*
+ * If you need to explain a lot you can use a comment block. Notice the
+ * single * on the first line. Comment blocks don't need to be three
+ * lines long or three characters shorter than the previous line.
+ */
+```
+
+## Whitespace
+
+Statements should have to breathe. In general always add blank lines between statements, unless they're a sequence of single-line equivalent operations. This isn't something enforceable, it's a matter of what looks best in its context.
+
+```php
+// Good
+public function getPage($url)
+{
+    $page = $this->pages()->where('slug', $url)->first();
+
+    if (! $page) {
+        return null;
+    }
+
+    if ($page['private'] && ! Auth::check()) {
+        return null;
+    }
+
+    return $page;
+}
+
+// Bad: Everything's cramped together.
+public function getPage($url)
+{
+    $page = $this->pages()->where('slug', $url)->first();
+    if (! $page) {
+        return null;
+    }
+    if ($page['private'] && ! Auth::check()) {
+        return null;
+    }
+    return $page;
+}
+```
+
+```php
+// Good: A sequence of single-line equivalent operations.
+public function up()
+{
+    Schema::create('users', function (Blueprint $table) {
+        $table->increments('id');
+        $table->string('name');
+        $table->string('email')->unique();
+        $table->string('password');
+        $table->rememberToken();
+        $table->timestamps();
+    });
+}
+```
+
+Don't add any extra empty lines between `{}` brackets.
+
+```php
+// Good
+if ($foo) {
+    $this->foo = $foo;
+}
+
+// Also Good
+if ($foo) $this->foo = $foo
```

Comment:
Avoid using single line `if` statements without braces. It can lead to confusion and bugs if more lines are added in the future.